### ISSUE TITLE
When retrieving a user's following list, an error occurs with the cursor when the count reaches approximately 2300.

### ISSUE THREAD

**Issue by 0xfocu5** [imperatrona/twitter-scraper#34]
Here is test code。
when the count reaches approximately 2300, The cursor will repeatedly fetch data from the beginning of the list.

```
cursor := ""
userId:="1323413840235696130"
num := 0
for {
		users, cursor, err = scarper.FetchFollowingByUserID(userId, 200, cursor)
		if err != nil {
			return err
		}
		num += 50
		fmt.Println(num)
	}
```

**Comment by cmj** [comment]
At a glance, it seems the line here is just sending an empty string, which would make the requests start over.

https://github.com/imperatrona/twitter-scraper/blob/152a0a1c2c60df3812b3fd9cfcaf61905f68a80f/follows.go#L75C16-L75C18

(I'm not sure how to fix this, but worth looking into) 

Also, that endpoint isn't replying with 200 results anymore; only 50? So you're seeing rate limits much sooner. 

I'm still getting 200 off the old v1.1 endpoint `https://api.twitter.com/1.1/friends/list.json?count=200` which results in about 26.8K before the limits are hit. People following that many are rare, but was just curious.

**Comment by 0xfocu5** [comment]
> At a glance, it seems the line here is just sending an empty string, which would make the requests start over.乍一看，这里的行似乎只是发送一个空字符串，这将使请求重新开始。
> 
> https://github.com/imperatrona/twitter-scraper/blob/152a0a1c2c60df3812b3fd9cfcaf61905f68a80f/follows.go#L75C16-L75C18
> 
> (I'm not sure how to fix this, but worth looking into)（我不确定如何解决这个问题，但值得研究）
> 
> Also, that endpoint isn't replying with 200 results anymore; only 50? So you're seeing rate limits much sooner.此外，该终端节点不再回复 200 个结果;只有 50 个？因此，您可以更快地看到速率限制。
> 
> I'm still getting 200 off the old v1.1 endpoint `https://api.twitter.com/1.1/friends/list.json?count=200` which results in about 26.8K before the limits are hit. People following that many are rare, but was just curious.我仍然从旧的 v1.1 端点 `https://api.twitter.com/1.1/friends/list.json?count=200` 获得 200，这导致在达到限制之前大约有 26.8K。关注这么多的人很少见，但只是好奇。


Yes, now the replies are limited to a maximum of 50 nodes. I'm not quite sure what the situation is, but I hope you achieve a good outcome. Best wishes!
